test/transmuxer.test.js

```diff
@@ -93,7 +87,7 @@ QUnit.test('buffers partial packets', function() {
   var
     datas = [],
     partialPacket1 = new Uint8Array(187),
-    partialPacket2 =  new Uint8Array(189);
+  partialPacket2 = new Uint8Array(189);
```

Comment:
Meant to just remove the extra space around the `=` but I messed up the indentation while I was at it. Fixing.